updated readme (#3508)
RasaHQ/rasa: README.md

> # Rasa (former Rasa Core + Rasa NLU)

Reviewer: ```suggestion
# Rasa (formerly Rasa Core + Rasa NLU)
```

Nothing has changed yet in terms of usage, but we are in the process of simplifying
everything ahead of the next major release. 

> Rasa is anopen source machine learning framework to automate text-and voice-based conversations. With Rasa, you can build chatbots on:

Reviewer: ```suggestion
Rasa is an open source machine learning framework to automate text-and voice-based conversations. With Rasa, you can build chatbots on:
```

- Facebook Messenger
- Slack
- Microsoft Bot Framework
- Rocket.Chat
- Mattermost
- Telegram
- Twilio
- Your own custom conversational channels

> or voice assistants on:

Reviewer: ```suggestion
or voice assistants as:
```
